travis.yml

```diff
@@ -0,0 +1,23 @@
+language: dart
+
+dart:
+  - 2.0.0
+  - dev
+
+dart_task:
+  - test
```

Comment:
Comment this out – it'll likely fail since there are no tests – yet

Reply:
done